content/influxdb/v2.0/write-data/developer-tools/api.md

```diff
@@ -16,48 +16,41 @@ Use the `POST` request method and include the following in your request:
 | Organization         | Use the `org` query parameter in your request URL.       |
 | Bucket               | Use the `bucket` query parameter in your request URL.    |
 | Precision            | Use the `precision` query parameter in your request URL. |
-| API token | Use the `Authorization: Token <API token>` header.                   |
+| API token | Use the `Authorization: Token YOUR_API_TOKEN` header.                   |
 | Line protocol        | Pass as plain text in your request body.                 |
 
-#### Example API write request
+#### Send a write request
 
-Below is an example API write request using `curl`.
-The URL depends on the version and location of your InfluxDB 2.0 instance _(see [InfluxDB URLs](/{{< latest "influxdb" >}}/reference/urls/))_.
-
-To compress data when writing to InfluxDB, set the `Content-Encoding` header to `gzip`.
-Compressing write requests reduces network bandwidth, but increases server-side load.
+The URL in the examples depends on the version and location of your InfluxDB 2.0 instance _(see [InfluxDB URLs](/influxdb/v2.0/reference/urls/))_.
```

Comment:
So we should be updating the URLs automatically. We could same something like:

```
The URL in the examples depends on the version and location of your InfluxDB 2.0 instance. <a href="#" class="url-trigger">Customize URLs in examples</a>.
```

This will trigger the URL selector modal which also includes a link to the InfluxDB URLs doc.

Reply:
done